apps/strapi-cms/src/api/homepage/content-types/homepage/schema.json

```diff
@@ -55,6 +55,11 @@
         }
       },
       "component": "common.hero-slide"
+    },
+    "pinned_webinars": {
```

Comment:
Shouldn't this be called `pinnedWebinars` using camelCase like all the others attributes?

Reply:
Or just `webinars`